services/libs/data-access-layer/src/old/apps/profiles_worker/orgs.ts

```diff
@@ -0,0 +1,26 @@
+import { DbStore } from '@crowd/database'
+
+export async function findMemberIdsInOrganization(
+  db: DbStore,
+  organizationId: string,
+): Promise<string[]> {
+  let result: { memberId: string }[]
+  try {
+    result = await db.connection().any(
+      `
+      select distinct "memberId" from "memberOrganizations"
```

Comment:
doesn't `memberOrganizations` have `deletedAt` column that we need to check here?